trl/trainer/online_dpo_trainer.py

```diff
@@ -92,23 +97,22 @@ class OnlineDPOTrainer(Trainer):
 
     def __init__(
         self,
-        model: Union[PreTrainedModel, nn.Module] = None,
-        ref_model: Union[PreTrainedModel, nn.Module] = None,
-        reward_model: Optional[nn.Module] = None,
+        model: Union[PreTrainedModel, nn.Module, None] = None,
```

Comment:
In what circumstances could the model be `None`?

Reply:
could be none when a pretrained model string id is given?

Reply:
See https://github.com/huggingface/trl/issues/2014

Reply:
> https://github.com/huggingface/trl/pull/2041#discussion_r1758452282

So `model` would be a string, not a `None`. Anyway passing a string is not support in Online DPO (do we need it?)

Reply:
ah yes you are right.. i was thinking in the script not the trainer... i don't think we need it

Reply:
let's make the model a required arg then